pkg/sosreport/index.html

```diff
@@ -28,20 +28,124 @@
+
+                                <label class="control-label" for="sos-skip-plugins">Skip These Plugins</label>
+                                <input id="sos-skip-plugins" class="form-control" title="List of plugin names to skip, comma delimited">
+
+                                <label class="control-label" for="sos-only-plugins">Only Run These Plugins</label>
+                                <input id="sos-only-plugins" class="form-control" title="List of plugin names to ONLY run, comma delimited">
+
+                                <label class="control-label" for="sos-plugin-option">Plugin Options</label>
+                                <input id="sos-plugin-option" class="form-control" title="Set plugin options, comma delimited list. Options take the form <plugin>.<option>=<value>">
+
+                                <label class="control-label" for="sos-allow-system-changes">Allow System Changes &nbsp;&nbsp;
+                                    <span class="pficon pficon-help" title="Certain system changes may be made such as loading kernel modules"></span>
+                                </label>
+                                <label class="checkbox-inline">
+                                    <input id="sos-allow-system-changes" type="checkbox">
+                                </label>
+
+                                <label class="control-label" for="sos-all-logs">Collect ALL Logs &nbsp;&nbsp;
+                                    <span class="pficon pficon-help" title="Collect all logs (and rotated logs) without sizelimit (WILL significantly increase archive size), overrides Log Size Limit"></span>
+                                </label>
+                                <label class="checkbox-inline">
+                                    <input id="sos-all-logs" type="checkbox">
+                                </label>
+
+                                <label class="control-label" for="sos-no-postproc">Skip Post Processing &nbsp;&nbsp;
+                                    <span class="pficon pficon-help" title="Do not have plugins perform post processing, such as removing passwords"></span>
+                                </label>
+                                <label class="checkbox-inline">
+                                    <input id="sos-no-postproc" type="checkbox">
+                                </label>
+
+                                <label class="control-label pf-c-switch" for="sos-verify">Verify Packages &nbsp;&nbsp;
+                                    <span class="pficon pficon-help" title="Have the package manager verify all installed packages"></span>
+                                </label>
+                                <label class="checkbox-inline">
+                                    <input id="sos-verify" type="checkbox">
+                                </label>
+
+                                <label class="control-label" for="sos-verbose">Verbose Logging &nbsp;&nbsp;
```

Comment:
those ` &nbsp;&nbsp;` will definitely get broken with translations. Please use css for adding space between components.